.github/scripts/publish

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -eux
+
+set +ux
```

Comment:
Could you confirm about what this is doing here? 

From this [resource](https://askubuntu.com/questions/1167965/what-does-set-x-do-in-a-bash-script), I understand that `set +x` disables the individual printing of commands/arguments when the code is being executed. So basically hiding this information from the command line. Where later on `set -x` enables it again.


This [article](https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html), talks about treating unset variables/parameters as an error for the `set +u` command.